packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -53,6 +53,10 @@ export function AddComment( {
 		return null;
 	}
 
+	// Generate contextual label for new comment
+	// For new comments, use a generic label since we don't have an ID yet
+	const commentLabel = _x( 'New Comment', 'Add new comment label' );
```

Comment:
```suggestion
	const commentLabel = __( 'New Comment' );
```

The code comments and context seem redundant to me.